3rdparty/org/scala-lang/BUILD

```diff
@@ -1,7 +1,7 @@
 def make_lib(name):
   jar_library(name=name,
     jars=[
-      jar(org='org.scala-lang', name=name, rev='2.11.11')
```

Comment:
Can you just nuke this file? pants should be pulling some default version of all these things, and then it would pick the right scala version too. If that works, we can do it in aiq repo too.

Reply:
yeah

Reply:
nice, so if we wanna go back to 2.11, we can just change the one config in pants.toml